tests/models/test_hparams.py

```diff
@@ -60,7 +60,7 @@ class SaveHparamsDecoratedModel(BoringModel):
     """ Tests that a model can take an object """
     @decorate
     @decorate
-    def __init__(self, hparams, *my_args, **my_kwargs):
+    def __init__(self, hparams, *_, **__):
```

Comment:
The argument names for the hyperparameter tests here in this file needs to stay like it is, please don't change it.

Reply:
do you have any idea how to deal with inherited API which is not used in this class?
if we would need to skip/ignore all then this Volture does not have much sense...
@aniketmaurya mind check if there is an option to deal in inherited API?

Reply:
@borda Another way is that we can either rename the variable to start with underscore (**_kwargs), Vulture will ignore these variables. Or we can add these to whitelist.

Reply:
I think training step variables should not be renamed. Then maybe we can ignore these names by maintaining [`ignore_names`](https://github.com/jendrikseipp/vulture#configuration) list.
We can keep something like this in `pyproject.toml`:
`ignore_names = ["batch_idx", "optimizer_idx", "batch"]`

And for variables like `args` and `kwargs` we can use underscore notation `*_args` and `**_kwargs`.

Reply:
@awaelchli how do you feel about `*_args` and `**_kwargs`?

Reply:
no matter what the renaming is, the tests must pass and they currently don't, because in the tests it matters what the var names are even if the tool says it is unused, it is actually used!
the dead code checker probably only looks at the static source files, but it can not determine such logic here.